starter_code/models/movies.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const movieSchema = new Schema({
+    title: String,
+    director: String,
+    stars: [String],
+    image: String,
+    description: String,
+    showTimes: [String]
```

Comment:
```suggestion
    showtimes: [String]
```

Como vocês podem observar nos objetos dos filmes, a propriedade `showtimes` é escrita toda com letra minuscula. 
Como o `seed` está diferente do model, nenhum `showtime` foi adicionado do banco de dado.